using_images/xpaas_images/a_mq.adoc

```diff
@@ -42,7 +42,7 @@ xref:configuring-params[Application Template Parameters].
 ** using the S2I (Source-to-image) tool, as described in
 xref:configuring-sti[Configuration Using S2I].
 
-ifdef::openshift-enterprise,openshift-origin[]
+ifdef::openshift-origin[]
```

Comment:
@tpoitras The xPaaS topics are only included in the openshift-online,openshift-enterprise,openshift-dedicated distros in the build_cfg anyway, so any openshift-origin ifdefs within the topics won't matter. Really these could actually be openshift-enterprise ifdefs, that just say that they were installed by default during installation, and link to the new default_imagestreams_templates topic (and that way there's no link or mention of installation in the dedicated or online builds, too).